Main.lean

```diff
@@ -57,6 +58,9 @@ def buildRun (p : Cli.Parsed) : IO UInt32 := do
     IO.eprintln "Run `yatima build -h` for further information."
     return 1
 
+instance : Coe String (Option String) where
+  coe := some
```

Comment:
Why is this needed after updating the dependencies?
@winston-h-zhang do you understand this?